src/Form/selectors.js

```diff
@@ -1,41 +1,40 @@
 const EMAIL_SELECTOR = `
-    input:not([type])[name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=""][name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=text][name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input:not([type])[id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=""][id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=text][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=""][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=email]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
-    input[type=text][aria-label*=mail i],
-    input:not([type])[aria-label*=mail i],
-    input[type=text][placeholder*=mail i]:not([readonly]),
-    input[autocomplete=email]:not([readonly]):not([hidden]):not([disabled])
-`
+input:not([type])[name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=""][name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=text][name*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input:not([type])[id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=""][id*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=text][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=""][placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input:not([type])[placeholder*=mail i]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=email]:not([readonly]):not([disabled]):not([hidden]):not([aria-hidden=true]),
+input[type=text][aria-label*=mail i],
+input:not([type])[aria-label*=mail i],
+input[type=text][placeholder*=mail i]:not([readonly]),
+input[autocomplete=email]:not([readonly]):not([hidden]):not([disabled])`
```

Comment:
Just reformatted.

Reply:
👀

Reply:
Yeah, I know 🙂. Initially we targeted only email fields and this allowed us to avoid a lot of DOM processing in js. Now the thing is vastly more complex elsewhere and we could probably simplify these selectors.